simple_history/tests/tests/test_signals.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import unicode_literals
+
+from datetime import datetime
+
+from django.test import TestCase
+
+from simple_history.signals import (
+    pre_create_historical_record,
+    post_create_historical_record,
+)
+
+from ..models import Poll
+
+today = datetime(2021, 1, 1, 10, 0)
+
+
+class PrePostCreateHistoricalRecordSignalTest(TestCase):
+    def setUp(self):
+        self.signal_was_called = False
+        self.signal_instance = None
+        self.signal_history_instance = None
+
+    def test_pre_create_historical_record_signal(self):
+        def handler(sender, instance, **kwargs):
+            self.signal_was_called = True
+            self.signal_instance = instance
+        pre_create_historical_record.connect(handler)
+
+        p = Poll(question="what's up?", pub_date=today)
+        p.save()
+
+        self.assertTrue(self.signal_was_called)
+        self.assertEqual(self.signal_instance, p)
+
+    def test_post_create_historical_record_signal(self):
```

Comment:
It's not getting the correct class. Do you know any other way I could do this? Since the manager is used to create the object, I thought that `manager.model.__class__` would work, but instead it's returning the Django Model class.